src/main/java/io/orkes/conductor/client/ApiClient.java

```diff
@@ -111,6 +111,9 @@ public ApiClient(String basePath, SecretsManager secretsManager, String keyPath,
     }
 
     public ApiClient(String basePath, String keyId, String keySecret) {
+        if(basePath == null) {
+            basePath = "http://localhost:8080/api";
```

Comment:
So default to localhost?

Reply:
yes.

Reply:
this has been there always when you init without any args.